extensions/scheduler/runtime/src/test/java/io/quarkus/scheduler/runtime/util/SchedulerUtilsTest.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.scheduler.runtime.util;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class SchedulerUtilsTest {
+
+    @Test
+    void testIsConfigValueCurlyBracesSyntax() {
```

Comment:
Would it make sense to test other `SchedulerUtils` methods as well?

Reply:
I added a few more. Is there something specific you'd like to have tested?

Reply:
Nope, this is perfect ;-).